jenkins/spark-nightly-build.sh

```diff
@@ -49,6 +50,7 @@ function distWithReducedPom {
 
         deploy)
             mvnCmd="deploy:deploy-file"
+            mvnExtaFlags="-Durl=$SERVER_URL -DrepositoryId=$SERVER_ID"
```

Comment:
@Gera Shegalov  We already have this URL defined in Jenkins as below instead, similar referrence in [settings.xml](https://github.com/NVIDIA/spark-rapids/blob/branch-22.02/jenkins/settings.xml#L105)

`-Durl=${URM_URL}-local -DrepositoryId=snapshots`

just change to
`mvnExtaFlags="-Durl=${URM_URL}-local -DrepositoryId=snapshots"`
from
`mvnExtaFlags="-Durl=$SERVER_URL -DrepositoryId=$SERVER_ID"`
BTW, I did not found where this `mvnExtaFlags`  been used in the deloy:deploy-file CLI in PR4371?